critiquebrainz/frontend/views/mapping.py

```diff
@@ -116,8 +116,11 @@ def spotify_confirm():
 
     if request.method == 'POST':
         # TODO(roman): Check values that are returned by add_mapping (also take a look at related JS).
-        mbspotify.add_mapping(release_group_id, 'spotify:album:%s' % spotify_id, current_user.id)
-        flash.success(gettext("Spotify mapping has been added!"))
+        res, _ = mbspotify.add_mapping(release_group_id, 'spotify:album:%s' % spotify_id, current_user.id)
+        if res:
+            flash.success(gettext("Spotify mapping has been added!"))
+        else:
+            flash.error(gettext("Could not add Spotify mapping!"))
```

Comment:
would prefer it if we logged the error as well here.

```suggestion
            flash.error(gettext("Could not add Spotify mapping!"))
            current_app.logger.error('Error while blah blah blah...')
```